modules/ballerina-test/src/test/java/org/ballerinalang/test/connectors/EndpointConnectorTest.java

```diff
@@ -153,8 +153,16 @@ public void testConnectorNegativeCases() {
                 91, 12);
         BAssertUtil.validateError(resultNegative, 8, "incompatible types: expected 'endpoint<Foo>', found 'string'",
                 96, 9);
-        BAssertUtil.validateError(resultNegative, 9, "unreachable code", 48, 5);
-        BAssertUtil.validateError(resultNegative, 10, "this function must return a result", 42, 1);
+        BAssertUtil.validateError(resultNegative, 9, "incompatible types: expected 'any', found 'endpoint<Foo>'",
```

Comment:
We need a better error message for this. when it says, it implies, the endpoint is a type, which is not.